### DISCUSSION TITLE
Osmand usable in Android Auto?

### DISCUSSION THREAD

**Discussion by wehkah** [osmandapp/OsmAnd#11447] [General]
Hello!

How likely is it that Osmand will be usable in Android Auto as a routing app instead of Google Maps in the nearest future? 

I am aware that an app has to fulfil some requirements to be accepted by Google as an AA app. But I also read that Google is opening its ecosystem to more and more third party apps. It really would be wonderful if Osmand could be one of these apps!

**Comment by polarbearing** [comment]
Duplicate of #3391

**Reply by wehkah** [reply]
> Duplicate of #3391

Sorry, I didn't realize that old thread being still alive!
